website/src/pages/auth/signin.tsx

```diff
@@ -183,7 +195,9 @@ const EmailSignInForm = ({
           variant="outline"
           size="lg"
           placeholder="Email Address"
-          {...register("email")}
+          {...register("email", { required: true, pattern: /[^\s@]+@[^\s@]+\.[^\s@]+/g })}
+          isInvalid={errors.email ? true : false}
+          errorBorderColor="orange.600"
```

Comment:
Can you use the `FormHelperText` some where around here and remove the `throwError` callback? see [https://chakra-ui.com/docs/components/form-control](https://chakra-ui.com/docs/components/form-control)